lib/symbol-store.ts

```diff
@@ -23,6 +23,10 @@
 // POSSIBILITY OF SUCH DAMAGE.
 
 export class SymbolStore {
+    private readonly uniqueSymbols: Record<string, string>;
+    private sortedSymbols: [string, string][];
+    private isSorted: boolean;
+
     constructor() {
         this.uniqueSymbols = {};
         this.sortedSymbols = [];
```

Comment:
Could probably get rid of these lines and initialize the members when they're declared